app/src/main/res/values-ru/strings.xml

```diff
@@ -94,17 +94,17 @@
 
         It offers exclusive extras like Search and showing Album Art, a sleep timer and many other settings. This elegant music player allows you creating and managing handy playlists as you desire.
 
-        Playlists are fully supported. You can add either individual files, or whole folders. If you won\'t like a file, you can either just remove it from a playlist, or delete the actual file altogether.
+        Полная поддержка плейлистов. You can add either individual files, or whole folders. If you won\'t like a file, you can either just remove it from a playlist, or delete the actual file altogether.
```

Comment:
how about translating the whole long app description? Not sure why you picked just a few parts of it :) Thanks